pkg/runner/run_context.go

```diff
@@ -96,6 +97,36 @@ func (rc *RunContext) startJobContainer() common.Executor {
 			binds = append(binds, fmt.Sprintf("%s:%s%s", rc.Config.Workdir, rc.Config.Workdir, bindModifiers))
 		}
 
+		if rc.Config.ContainerArchitecture == "" {
+			rc.Config.ContainerArchitecture = fmt.Sprintf("%s/%s", "linux", runtime.GOARCH)
+		}
```

Comment:
This seems like old code re-introduced through merge

Reply:
```suggestion
```